wondrous-app/components/Leaderboard/LeaderboardUserRowTasks.tsx

```diff
@@ -0,0 +1,116 @@
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import format from 'date-fns/format';
+import { useRouter } from 'next/router';
+import palette from 'theme/palette';
+import { PRIVATE_TASK_TITLE } from 'utils/constants';
+import { cutString } from 'utils/helpers';
+import useMediaQuery from 'hooks/useMediaQuery';
+import { useReducer } from 'react';
+
+const LeaderboardUserRowTasks = ({ contributorTask }) => {
+  const router = useRouter();
+  const { isTabletScreen } = useMediaQuery();
+  const [taskCount, setTaskCount] = useReducer((state) => state + 10, 10);
+  const contributorTasks = contributorTask?.tasks;
+  const tasks = contributorTasks?.slice(0, taskCount) || [];
+  const showMoreButton = tasks.length < contributorTasks.length;
+  return (
+    <Grid container item width="100%" borderRadius="0 0 6px 6px">
+      <Grid container item width="100%" display="flex" gap="14px" padding="14px">
+        {tasks.map((task) => {
+          const podName = task?.podName || task?.pod?.name;
+          const podColor = task?.podColor || task?.pod?.color || palette.grey78;
+          const privateTask = task?.title === PRIVATE_TASK_TITLE;
+          const handleOnClickTask = (e) => {
```

Comment:
can we use Link component? the reason is so we can have 'Click to open in a new tab'

Reply:
Updated.